scripts/e2e-custom.sh

```diff
@@ -435,6 +491,17 @@ test_exec() {
 
     test_chat
 
+    # Test RAG capabilities
+    if [ "$DEV_PREVIEW_FEATURE_GATING" -eq 1 ]; then
+      test_convert_taxonomy_for_rag
```

Comment:
we probably need to run `test_taxonomy 3` before these steps, so the taxonomy is initialized with knowledge data (tests 1 and 2 have compositional skills, instead)